src/components/navbar/navbar.module.scss

```diff
@@ -0,0 +1,77 @@
+.navbar {
+  height: 80px;
+  margin-left: 16px;
+  display: flex;
+  justify-content: start;
+  align-items: center;
+}
+
+.menu-bars {
+  margin-left: 2rem;
+  font-size: 2rem;
+  background: none;
+}
+
+.nav-menu {
+  background-color: #3d3f46;
+  width: inherit;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  position: fixed;
+  top: 0;
+  left: -100%;
+  transition: 850ms;
+
+  &.active {
+    left: 0;
+    transition: 350ms;
+  }
+}
+
+.nav-text {
+  display: flex;
+  justify-content: start;
+  align-items: center;
+  padding: 8px 0px 8px 16px;
+  list-style: none;
+  height: 60px;
+
+  a {
+    text-decoration: none;
+    color: #f5f5f5;
+    font-size: 18px;
+    width: 95%;
+    height: 100%;
+    display: flex;
+    align-items: center;
+    padding: 0 16px;
+    border-radius: 4px;
+
+    &:hover {
+      background-color: #1a83ff;
+    }
+  }
+}
+
+.nav-menu-items {
+  width: 100%;
+  margin-left: 0px;
+}
+
+.navbar-toggle {
+  background-color: #3d3f46;
+  width: 100%;
+  height: 52px;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+ul {
+  padding-inline-start: 18px;
+}
+
+span {
+  margin-left: 8px;
+}
```

Comment:
Do these actually have an effect?

If they aren't actually affecting the layout, we should remove them.

If they are then they must be applied globally, which I'd prefer not to do in this app. We could just move them inside of a .classname in that case.